bin/rialto/runtime/src/lib.rs

```diff
@@ -991,10 +1002,11 @@ mod tests {
 			core::mem::size_of::<pallet_bridge_messages::Call<Runtime>>() <=
 				BRIDGES_PALLETS_MAX_CALL_SIZE
 		);
-		// Largest inner Call is `pallet_session::Call` with a size of 224 bytes. This size is a
-		// result of large `SessionKeys` struct.
-		// Total size of Rialto runtime Call is 232.
-		const MAX_CALL_SIZE: usize = 232;
+		// Largest inner Call is `pallet_bridge_beefy::Call` with a size of 240 bytes. This size
+		// is a result of the large number and complexity of the structs received by the
+		// `submit_commitment` call.
+		// Total size of Rialto runtime Call is 280.
+		const MAX_CALL_SIZE: usize = 280;
```

Comment:
It is better to avoid increasing this constant when possible, especially when it is increased by 20%. Would be better to make your large args be `Box<T>` instead of `T`. Don't know why this particular value has been chosen (it is used by Polkadot/Kusama/..), but I guess there's some research+assumptions behind it.

Reply:
Ok, sorry I didn't know that. For the moment I boxed `MmrLeaf`. It was taking 96 bytes.